### PR TITLE
Add Bruno Catteau

### PR DESCRIPTION
**Pull request by brunocatteau**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13567,6 +13567,68 @@ var data = {
...
       }
+    },
+    {
+      "since": "2016-09-23",
+      "name": "Bruno CATTEAU",
+      "bio": "Associé chez Lucca, je suis responsable de l'infrastructure Saas et de la sécurité. Je suis fan d'automatisation, dans le domaine de l'informatique mais aussi dans la vraie vie. Ma deuxième passion est le bricolage.",
+      "picture": "https://avatars2.githubusercontent.com/u/2759474",
+      "websites": [
+        {
+          "name": "Lucca",
```
Comment (dadoonet) [inline baggers.js]: Please look at the README (Websites) and fix accordingly,

File: baggers.js [hunk 2]
```diff
@@ -13567,6 +13567,68 @@ var data = {
...
+          "url": "https://www.lucca.fr/"
+        },
+         {
+          "name": "Brico-info.com",
```
Comment (dadoonet) [inline baggers.js]: Same here

**Review by dadoonet — CHANGES_REQUESTED** [review]

**Review by brunocatteau — COMMENTED** [review]
Fixed

**Comment by dadoonet** [comment]
Hi @brunocatteau 

Could you please rebase on latest master (possibly squash your commits and force push to your repo? So I can merge it?
There is a conflict change for now.

**Comment by dadoonet** [comment]
Ping?

**Comment by dadoonet** [comment]
@brunocatteau Can you rebase on master please?

File: baggers.js [hunk 3]
```diff
@@ -13567,6 +13567,68 @@ var data = {
         "twitter": "GMouron",
         "mail": "gmouron@lucca.fr"
       }
+    },
+    {
+      "since": "2016-09-23",
+      "name": "Bruno CATTEAU",
```
Comment (dadoonet) [inline baggers.js:13573]: Can you replace with `Bruno Catteau` so it's more consistent with others?

**Comment by dadoonet** [comment]
Ping?

**Comment by dadoonet** [comment]
Pas de News. Closing.

**Pull request closed without merging**